t/server/metadata_suite.c

```diff
@@ -0,0 +1,81 @@
+#include <mpi.h>
+
+#include "unifycr_configurator.h"
+#include "unifycr_metadata.h"
+#include "unifycr_log.h"
+#include "unifycr_runstate.h"
+
+#include "t/lib/tap.h"
+#include "t/lib/testutil.h"
```

Comment:
It doesn't look like you're using any functions from `t/lib/testutil.h`, so can probably omit this. These are convenience functions that Ned originally wrote.